### ISSUE TITLE
"Could not load generator generators/data_mapper/devise_generator" when using Devise Datamapper generator

### ISSUE THREAD

**Issue by luma** [jm81/dm-devise#3]
Hi,

I get the following error whenever I use the Devise Datamapper generator:

```
$ rails generate data_mapper:devise User
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0.rc/lib/rails/script_rails_loader.rb:11: warning: Insecure world writable dir /usr/local/bin in PATH, mode 040777
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/bundler-1.0.0.rc.1/lib/bundler/runtime.rb:132: warning: Insecure world writable dir /usr/local/bin in PATH, mode 040777
[WARNING] Could not load generator "generators/data_mapper/devise_generator". Error: DataMapper is not missing constant Generators!.
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:479:in `load_missing_constant'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:183:in `const_missing'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:181:in `each'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:181:in `const_missing'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/dm-devise-1.1.2/lib/generators/data_mapper/devise_generator.rb:6
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:239:in `require'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:239:in `require'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:225:in `load_dependency'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:591:in `new_constants_in'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:225:in `load_dependency'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:239:in `require'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/generators.rb:286:in `lookup'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/generators.rb:282:in `each'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/generators.rb:282:in `lookup'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/generators.rb:281:in `each'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/generators.rb:281:in `lookup'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/generators.rb:141:in `find_by_namespace'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/generators.rb:161:in `invoke'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/commands/generate.rb:10
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:239:in `require'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:239:in `require'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:225:in `load_dependency'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:591:in `new_constants_in'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:225:in `load_dependency'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/activesupport-3.0.0/lib/active_support/dependencies.rb:239:in `require'
[PATH REMOVED]/.rvm/gems/ruby-1.8.7-p299/gems/railties-3.0.0/lib/rails/commands.rb:17
script/rails:6:in `require'
script/rails:6
```

My Environment is:
- ruby 1.8.7 p299
- Rails 3
- Datamapper 1.0.2
- dm-devise 1.1.2
- Devise 1.1.2

**Comment by jm81** [comment]
Fixed by commit 9aac55a174125863dd73d67e465de5ca42947aba . I'll push out a new version of the gem shortly. Let me know if that doesn't fix it for you.

**Comment by luma** [comment]
That fixed it! Thanks for taking the time to look at it :-)
